graphics/managed_surface.cpp

```diff
@@ -486,15 +484,25 @@ void transBlitPixel(TSRC srcVal, TDEST &destVal, const Graphics::PixelFormat &sr
 		rDest = rSrc;
 		gDest = gSrc;
 		bDest = bSrc;
+		aDest = 0xff;
 	} else {
 		// Partially transparent, so calculate new pixel colors
+		destFormat.colorToARGB(destVal, aDest, rDest, gDest, bDest);
 		double alpha = (double)aSrc / 255.0;
 		rDest = static_cast<byte>((rSrc * alpha) + (rDest * (1.0 - alpha)));
 		gDest = static_cast<byte>((gSrc * alpha) + (gDest * (1.0 - alpha)));
 		bDest = static_cast<byte>((bSrc * alpha) + (bDest * (1.0 - alpha)));
```

Comment:
Looks like these 4 lines are being replaced by what's below

Reply:
Yes, totally, I kept then in while writing the new ones, and then forgot to remove them.

Reply:
And this is now fixed.